services/storage/src/simcore_service_storage/s3_client.py

```diff
@@ -244,20 +244,27 @@ async def get_file_metadata(
 
     @s3_exception_handler(log)
     async def copy_file(
-        self, bucket: S3BucketName, src_file: SimcoreS3FileID, dst_file: SimcoreS3FileID
+        self,
+        bucket: S3BucketName,
+        src_file: SimcoreS3FileID,
+        dst_file: SimcoreS3FileID,
+        bytes_transfered_cb: Optional[Callable[[int], None]],
     ) -> None:
         """copy a file in S3 using aioboto3 transfer manager (e.g. works >5Gb and creates multiple threads)
 
         :type bucket: S3BucketName
         :type src_file: SimcoreS3FileID
         :type dst_file: SimcoreS3FileID
         """
-        await self.client.copy(
+        copy_options = dict(
```

Comment:
MINOR: I always struggle to find the right doc for s3. Would you mind putting a link here to the reference doc for these copy options?

Reply:
hmm, if you install storage:
```
cd services/storage
make install-dev
```
and then hover on the copy function you will see this:
![image](https://user-images.githubusercontent.com/35365065/187854960-f9a66086-caca-4d0a-87b7-82ccea2265cd.png)

That is enough no?

Reply:
perfect! Thx for pointing thta out